frontend/src/app/modules/imports/components/modal_destination/import-modal-destination.component.ts

```diff
@@ -40,7 +40,7 @@ export class ImportModalDatasetComponent implements OnInit, OnDestroy {
   }
 
   onSubmit() {
-    this.importProcessService.beginProcess(this.selectDatasetForm.value);
+    this.importProcessService.beginProcess(this.selectDestinationForm.value);
```

Comment:
It would be beneficial to add error handling for the form submission. This could be done by checking if the form is valid before submitting and displaying appropriate error messages if it is not. [medium]